package.json

```diff
@@ -1,4 +1,18 @@
 {
+  "name": "claim_for_crown_court_defense",
+  "description": "Claim for Crown Court Defence",
+  "version": "1.0.0",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence.git"
+  },
+  "author": "LAA Developers",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/issues"
+  },
+  "homepage": "https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence#readme",
   "dependencies": {
```

Comment:
Can you also add "main:application.js" for completion?

Reply:
thanks @mathan11480-moj but we don't utilise an entry file in this app.

Reply:
👍 do we even ship this as an npm package?